ChangeLog.txt

```diff
@@ -1,8 +1,31 @@
 Next v0.1.X
 -----------
 
+Cultures:
+
+- Added namelist for Vandal culture
+- Added dynasty names for Roman, Romano-Gallic, Romano-Hispanic, Romano-Thracian, Romano-Illyrian and Romano-Aramean cultures
+- Added many new name equivalencies and fixed some existing ones
+
+Localization:
+- Added numerous cultural title localization for Britannia, Gallia, Italia, Hispania, Africa, Eastern Roman Empire, Persia, Pontic-Caspian steppe, Tamazgha and Germania Magna
+- Fixed some typos among the title names
+
+History:
+- Expanded Nepos' Dalmatia based on a source, which states that it included lands between Drina and Sava. In addition, Dalmatia is now controls Epidaurum instead of ERE
+- Some work on character cultures:
+   - Nepos and Glycerius were made Romano-Illyrian because they were natives to Dalmatia
+   - All the popes are now have appropriate cultures to reflect their origins
+   - Macrinus, Aemilian and some Roman families were made Romano-African
+- Made Burdigala Romano-Gallic
+- Expanded Syagrius' Gaul to include Reistete and Viromandia. This made because all the maps show that Syagrius' kingdom controlled some territories north of Durocorturum
+- Added lowborn provincial governors for Asia, Lycia, Lydia, Achaea and Olisipo
```

Comment:
Why lowborn?

Reply:
> Why lowborn?

Of course they are noble, but its not know if they were related to any aristocratic families. This wording was used by Erik W, who on that basis labelled some Roman consuls as lowborn. But yeah, it does not look very well and thus will be removed.

Reply:
Wasn't that the thing where you labelled them lowborn in history and the game automatically created a dynasty on start? Anyhow, we agree that it is confusing